packages/examples/cvat/recording-oracle/src/services/cloud/gcs.py

```diff
@@ -32,6 +32,11 @@ def remove_file(self, key: str, *, bucket: str | None = None) -> None:
         bucket_client = self.client.get_bucket(bucket)
         bucket_client.delete_blob(unquote(key))
 
+    def remove_files(self, keys: list[str], *, bucket: str | None = None):
+        bucket = unquote(bucket) if bucket else self._bucket
+        bucket_client = self.client.get_bucket(bucket)
+        bucket_client.delete_blobs([unquote(key) for key in keys])
```

Comment:
The same comment about using a filter instead. I'm pretty sure there is such an option.

Reply:
Not to my knowledge. 
See https://cloud.google.com/storage/docs/deleting-objects#delete-objects-in-bulk

Since we're not using GCS for uploads right now, it's not really an issue right now, but I've added a warning in case it's going to be used some time.

Reply:
Discussed, GCS wrapper can just provide an unoptimized version.